src/types/profile/settings.rs

```diff
@@ -38,7 +40,9 @@ impl Default for Settings {
             streaming_server_url: STREAMING_SERVER_URL.to_owned(),
             interface_language: "eng".to_owned(),
             audio_language: "eng".to_owned(),
+            secondary_audio_language: "".to_owned(),
```

Comment:
if those are optional lets use Option<String>

Reply:
I guess then we don't need a migration?

Reply:
seems like it, will test it